lib/shopify_api/resources/fulfillment_request.rb

```diff
@@ -0,0 +1,11 @@
+module ShopifyAPI
+  class FulfillmentRequest < Base
+    init_prefix :order
+
+    def order_id
+      @prefix_options[:order_id]
+    end
+
+    def cancel; load_attributes_from_response(post(:cancel, {}, only_id)); end
```

Comment:
I'm wondering if `cancel` should maybe be `cancel_fulfillment`? Reason being that fulfillment services have a single callback url that currently defines `fetch_tracking`, `fetch_stock_levels`, soon `fulfill` (with fulfillment requests), and now this `cancel`. So I think being explicit might make sense.